src/Core/Theme/EventSubscriber.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop-project.org/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Theme;
+
+use PrestaShop\PrestaShop\Adapter\Cache\Clearer\SymfonyCacheClearer;
+use PrestaShopBundle\Event\ThemeManagementEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class EventSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var SymfonyCacheClearer
+     */
+    private $cacheClearer;
+
+    /** @var bool */
+    private $cleared = false;
+
+    public function __construct(SymfonyCacheClearer $cacheClearer)
+    {
+        $this->cacheClearer = $cacheClearer;
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            ThemeManagementEvent::INSTALL => 'onThemeInstalledOrUninstalled',
+            ThemeManagementEvent::UNINSTALL => 'onThemeInstalledOrUninstalled',
+        ];
+    }
+
+    public function onThemeInstalledOrUninstalled(ThemeManagementEvent $event): void
+    {
+        if (!$this->cleared) {
```

Comment:
I don't think you need this since the `SymfonyCacheClearer` already handles this internally